mirobo/plug.py

```diff
@@ -0,0 +1,37 @@
+from .device import Device
+
+PROPERTY_POWER = 'power'
+PROPERTY_TEMPERATURE = 'temperature'
+PROPERTY_CURRENT = 'current'
+
+
+class Plug(Device):
+    """Main class representing the smart wifi socket / plug."""
+
+    def __init__(self, ip: str, token: str, start_id: int = 0, debug: int = 0) -> None:
+        super().__init__(ip, token, start_id, debug)
+        self.manual_seqnum = -1
```

Comment:
Is the `__init__` here unnecessary though, it just sets the `manual_seqnum` it never uses :-)